src/Sulu/Bundle/ContactBundle/Resources/public/js/components/accounts/components/content/main.js

```diff
@@ -17,7 +17,12 @@ define(function() {
                 url: '/admin/content-navigations?alias=account'
             },
             toolbar: {
-                template: 'default'
+                buttons: [
+                    'saveWithOptions',
+                    {'settings': {
+                        dropdownItems: ['delete']
+                    }}
```

Comment:
Im ok with the syntax in general. But what is the benefit of using the id as a fallback for the parent-property? We will just create two ways of achieving the same thing, which we tried to avoid in the first place. Morever your suggestion of the callback mirrors the possibility of passing a callback via the options, which already exists